cape.go

```diff
@@ -1,6 +1,19 @@
 package cli
 
+import "github.com/capeprivacy/attest/attest"
+
+type Checksums struct {
+	Input    []byte `json:"input"`
+	Function []byte `json:"function"`
+	Output   []byte `json:"output"`
+}
+
 type RunResult struct {
-	Type    string `json:"type"`
-	Message []byte `json:"message"`
+	Type          string    `json:"type"`
+	Message       []byte    `json:"message"`
+	Checksums     Checksums `json:"checksums"`
+	SignedResults []byte    `json:"signed_checksums"`
```

Comment:
What's the reason for the field name in the struct not matching the field name in the received json value?

Reply:
We changed the json names sort of late in the game and I missed this when updating, will fix :)